scripts/components/statssynchronizercomponent/StatsSynchronizerComponent.gd

```diff
@@ -291,12 +312,15 @@ func add_experience(amount: int):
 
 
 func apply_boost(boost: Boost):
-	hp_max += boost.hp_max
-	hp += boost.hp
-	attack_power_min += boost.attack_power_min
-	attack_power_max += boost.attack_power_max
+	for statName in boost.statBoostDict:
+		assert(get(statName) != null, "This property does not exist as a stat.")
```

Comment:
We should not be using asserts on the server side. An assert will close the exe if it fails. This is a costly thing in an online service. It is better to handle the error cases by ignoring requests.